org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -545,11 +545,25 @@
                   <iterate>
                      <adapt
                            type="org.eclipse.core.resources.IResource">
+                        <test
+                              forcePluginActivation="true"
+                              property="org.eclipse.wildwebdeveloper.isHTMLProject">
+                        </test>
                      </adapt>
                   </iterate>
                </with></enablement>
          </contextualLaunch>
       </shortcut>
+   </extension>
+    <extension
+         point="org.eclipse.core.runtime.adapters">
+      <factory
+            adaptableType="org.eclipse.core.resources.IResource"
+            class="org.eclipse.wildwebdeveloper.debug.firefox.FirefoxLaunchableAdapterFactory">
```

Comment:
This is not really related to firefox, as we'll probably have the same logic for Chrome.
It could be something like "HTMLLaunchableAdapterFactory"